test/powershell/Modules/Microsoft.PowerShell.Utility/Test-Json.Tests.ps1

```diff
@@ -172,7 +172,7 @@ Describe "Test-Json" -Tags "CI" {
         # Exactly one type should match
         $types = 'string', 'number', 'boolean', 'null', 'array', 'object'
         $types | Where-Object {
-            $schema = "{ 'type': '$_' }"
+            $schema = '{ "type": "$_" }'
```

Comment:
```suggestion
            $schema = "{ `"type`": `"$_`" }"
```

Reply:
The test is not failed after my fix.

Reply:
I'm confused.  If I could get an explanation on this outside of this PR, I'd be grateful.  In my extracted code it doesn't work unless I change `Where-Object` to `ForEach-Object`.